tools/sdk-generation-pipeline/pipelines/steps/generate-sdk.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  - name: readmeFile
+    type: string
+
+  - name: trigger
+    type: string
+
+steps:
+  - bash: |
+      publishResult \
```

Comment:
I assume this is posting results to our eventhub. Have we tried subscribing to DevOps webhook notifications instead of directly using the eventhub yet? cc @chunyu3